cran/paws.security.identity/R/sts_service.R

```diff
@@ -102,6 +102,7 @@ NULL
 #' @rdname sts
 #' @export
 sts <- function(config = list()) {
+  .sts$metadata$endpoints <- .get_sts_endpoints(config$profile)
```

Comment:
This felt a little dirty, but since `endpoints` is dependent on `profile` this seemed like the natural way to do it.